### DISCUSSION TITLE
Error saat coba run library leaflet

### DISCUSSION THREAD

**Discussion by sabrinaa311** [teamalgoritma/community#222] [Q&A] [answered]
![3_data_visualization-main - RStudio 24_07_2022 09_36_20](https://user-images.githubusercontent.com/107026622/180629881-8dcdd113-8dac-425c-ae45-44862a58a5a6.png)

itu kenapa bisa error seperti itu ya kak

**Comment by idoali** [comment, marked as answer]
Kemungkinan error tersebut dikarenakan alamat directory yang terlalu panjang. Contohnya, ibu bisa memindahkan directory kerja Ibu yang sebelumnya di `C:/Users/HP/Downloads/Algoritma...` menjadi `D:/`. 

Masalah ini adalah masalah yang umum saat menggunakan R di windows.

**Reply by sabrinaa311** [reply]
berartii working directorynya diganti ya kak?

**Reply by idoali** [reply]
Iya benar bu. Pastikan nama alamat directory tidak panjang
